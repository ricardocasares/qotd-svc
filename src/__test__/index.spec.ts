import { QOTD_DB, pick } from "../";

test("picks a random quote", () => {
  const quote = pick(QOTD_DB);
  expect(quote).toBeTruthy();
});

test("picks the only quote available", () => {
  const quote = pick(["A quote"]);
  expect(quote).toMatchSnapshot();
});
