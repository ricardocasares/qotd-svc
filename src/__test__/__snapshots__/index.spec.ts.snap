// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`picks the only quote available 1`] = `"A quote"`;
